spec/factories/execution_environment.rb

```diff
@@ -73,18 +73,18 @@
     created_by_teacher
     default_memory_limit
     default_cpu_limit
-    docker_image { 'openhpi/co_execenv_python:3.4' }
```

Comment:
Generally, I am fine with this change. However, please note that 3.4 is still supported by us in production, and some language-specific changes might cause incompatibilities between code written for 3.4 running with 3.8. Have you checked all seed files for compatibility with 3.8?

Reply:
Thank you for pointing this out. I guess the update was not strictly necessary. Being on the latest version might be nice for future course support. I have no issue reversing to 3.4 if you feel that is better.

> Have you checked all seed files for compatibility with 3.8?

Yes it works with the seed data.